components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -49,6 +49,8 @@ fun String.isEmail() = re.emailish.matches(this)
 
 fun String.isGeoLocation() = re.geoish.matches(this)
 
+fun String.toDisplayUrl() = URLStringUtils.toDisplayUrl(this)
```

Comment:
See comment above. This doesn't seem right. Wdyt?

Reply:
Agreed.  I'll move this.